src/components/legacy/markdown/code-block.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import { CopyIcon } from '../icons/markdown-icons/copy';
-import copy from 'copy-to-clipboard';
 import styled, { CSSProperties } from 'styled-components';
 import { onlyText } from '../../support/only-text';
+import { Copy } from '../../ui/copy/copy';
```

Comment:
Even though code-block is legacy code it's okay to add it. Let's tackle and eliminate the legacy folder soon afterwards instead.

Reply:
👍